app/templates/audio_settings.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %} {% block content %}
+<div class="container mt-4">
+    <h1>Audio Settings</h1>
+    <form action="{{ url_for('get_choices') }}" style="width: 200px; margin: autio" method="post" id="form1">
+        <div class="form-outline">
+            <label>Music:</label>
+            <input type="radio" id="Music_On" name="music_on" value="True" class="form-check-input">
+            <label for="Music_On" class="form-check-label">On</label>
```

Comment:
`music-on`, `music-off` - lowercase with dash between (fix in all applicable places)